src/twisted/mail/pop3.py

```diff
@@ -1545,7 +1568,9 @@ def _dispatch(self, command, default, *args):
         @param args: Arguments to the handler function.
         """
         try:
-            method = getattr(self, 'handle_'+command, default)
+            if isinstance(command, bytes):
+                command = command.decode("utf-8")
+            method = getattr(self, 'handle_' + command, default)
```

Comment:
Same as above, I picture this looking a lot like L600:
https://github.com/twisted/twisted/blob/8095f968910471e017e508244391b30e099cdd93/src/twisted/mail/pop3.py#L600